ecommerce/loadgen/generate_load.py

```diff
@@ -31,19 +31,24 @@
 # INSERT TEMPLATES
 item_insert = "INSERT INTO shop.items (name, category, price, inventory) VALUES ( %s, %s, %s, %s )"
 user_insert = "INSERT INTO shop.users (email, is_vip) VALUES ( %s, %s )"
-purchase_insert = "INSERT INTO shop.purchases (user_id, item_id, quantity, purchase_price, status) VALUES ( %s, %s, %s, %s )"
+purchase_insert = "INSERT INTO shop.purchases (user_id, item_id, quantity, purchase_price, status) VALUES ( %s, %s, %s, %s, %s )"
 
 
 # Initialize Kafka
-producer = KafkaProducer(
-    bootstrap_servers=[kafkaHostPort],
-    value_serializer=lambda x: json.dumps(x).encode("utf-8"),
-    security_protocol='SASL_SSL',
-    sasl_mechanism='PLAIN',
-    sasl_plain_username=os.getenv("CONFLUENT_API_KEY", ""),
-    sasl_plain_password=os.getenv("CONFLUENT_API_SECRET", "")
-)
-
+if os.getenv("CONFLUENT_API_KEY", "") == "" or os.getenv("CONFLUENT_API_SECRET", "") == "":
```

Comment:
Nit: I think you can get away without the default values with `getenv` and just let it return None if the env variables are not set
```
if os.getenv("CONFLUENT_API_KEY") and os.getenv("CONFLUENT_API_SECRET"):
    producer = KafkaProducer(
        bootstrap_servers=[kafkaHostPort],
        value_serializer=lambda x: json.dumps(x).encode("utf-8"),
        security_protocol='SASL_SSL',
        sasl_mechanism='PLAIN',
        sasl_plain_username=os.getenv("CONFLUENT_API_KEY"),
        sasl_plain_password=os.getenv("CONFLUENT_API_SECRET")
    )
else:
    producer = KafkaProducer(
        bootstrap_servers=[kafkaHostPort],
        value_serializer=lambda x: json.dumps(x).encode("utf-8"),
    )
```

Reply:
Good suggestion! Just updated this!